floris/turbine_library/turbine_previewer.py

```diff
@@ -130,25 +130,25 @@ def  power_curve(
         """
         shape = (1, wind_speeds.size, 1)
         if self.turbine.multi_dimensional_cp_ct:
-            power_interps = {
+            power_functions = {
                 k: multidim_power_down_select(
-                    np.full(shape, self.turbine.power_interp),
+                    np.full(shape, self.turbine.power_function),
                     dict(zip(self.turbine.condition_keys, k)),
                 )
-                for k in self.turbine.power_interp
+                for k in self.turbine.power_function
```

Comment:
Just a quick note before the weekend - I think this is an incorrect change. This iterates on `power_function` but it isn't utterable.

Reply:
Thanks for calling this out @rafmudaf. I didn't look at turbine_previewer.py---I was thinking I'd open a seperate bugfix for that at some point. This change probably got caught up in a usual variable name swap and I didn't notice.